frontend/src/views/ProjectView.vue

```diff
@@ -1,30 +1,131 @@
 <template>
-  <ul>
-    <li v-for="[key, file] in files" :key="key">
-      {{ `${file.file.filename} ${file.uploadedSize} ${file.statusCode} ${file.statusText}` }}
-    </li>
-  </ul>
-  <!--TODO: It's a place holder and need to be refactored -->
-  <DragToStartField @dropFilesHandler="dropFilesHandler" />
+  <div class="card m-5">
+    <div class="card-body">
+      <div
+        class="card-title d-flex justify-content-between align-items-baseline"
+        :class="{ 'border-bottom': isShowed }"
+      >
+        <h4>
+          <button type="button" class="btn text-secondary" @click.prevent="isShowed = !isShowed">
+            <font-awesome-icon v-if="isShowed" :icon="['fas', 'minus']" />
+            <font-awesome-icon v-else :icon="['fas', 'angle-down']" />
+          </button>
+          <span style="color: var(--main-color)">Upload Files:</span>
+        </h4>
+        <div class="align-bottom fw-bold text-secondary">
+          {{
+            humanifyFileSize(
+              Array.from(files.values()).reduce((pre, element) => pre + element.file.filesize, 0)
```

Comment:
Your component should provide a computed value here. You don't want this kind of things in your template.